tests/telemetry/test_telemetry.py

```diff
@@ -133,3 +157,16 @@ def test_telemetry_ouput(duthosts, rand_one_dut_hostname, ptfhost, localhost):
     result = str(show_gnmi_out)
     inerrors_match = re.search("SAI_PORT_STAT_IF_IN_ERRORS", result)
     pytest_assert(inerrors_match is not None, "SAI_PORT_STAT_IF_IN_ERRORS not found in gnmi_output")
+
+def test_osbuild_version(duthosts, rand_one_dut_hostname, ptfhost, localhost):
```

Comment:
I am wondering if this test case need to setup telemetry like other test cases. For example `test_sysuptime` in PR #2583.
https://github.com/Azure/sonic-mgmt/pull/2583/files#diff-c0500c77703069eaa61a0c9b61f11d8bc9852794558c9efcd58e0b22d993c51cR141
Fixture `setup_streaming_telemetry` is to setup telemetry on DUT.

Reply:
It is not necessary as the test order. But it is also harmless as it already checks if the setup (restarting the service in non-cert mode). I'll include that step in the next iteration.